browser_use/browser/browser.py

```diff
@@ -18,6 +18,8 @@
 
 logger = logging.getLogger(__name__)
 
+CAMOUFOX = True
+
```

Comment:
```suggestion
CAMOUFOX = True

```

this should be an env var instead of hardcoded

Reply:
ah ok no worries. Support for non-chromium browsers did just get merged recently https://github.com/browser-use/browser-use/pull/950, so there may be some interest in this. I suspect it's too much work to get this working properly though unless you're willing to put in weeks of work. There are tons of raw CDP calls all over the codebase that would need to be switched to webdriver bidi equivalents without breaking anything.